### PR TITLE
[Automate 2940] remove v1 annotations

### PR DESCRIPTION
**Pull request by msorens**
### :nut_and_bolt: Description: What code changed, and why?

Removing IAM v1 annotations from all proto files since they are no longer used.

99% of the work was done with these commands; commits are separate so you could see what each of these did.
```
# Strip single-line annotations
find . -name '*.proto' -exec sed -i '' \
    '/option (chef.automate.api.policy)\./d' {} \;

# Strip multiple-line annotations
find . -name '*.proto' -exec sed -i '' \
    '/option (chef.automate.api.policy) *= *{/,/};/d' {} \;

# Strip import and related comments
find . -name '*.proto' -exec sed -i '' \
    '/for option (chef.automate.api.policy)/,/for option (chef.automate.api.iam.policy)/d' {} \;
```
Just a bit of manual cleanup was needed besides that.

### :chains: Related Resources
NA

### :+1: Definition of Done
All `chef.automate.api.policy` annotations are removed.
All remaining annotations use the v2 variation, `chef.automate.api.iam.policy`.

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] Tests added/updated?
- [ ] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: api/external/applications/applications.proto [hunk 1]
```diff
@@ -9,9 +9,6 @@ import "protoc-gen-swagger/options/annotations.proto";
 // TODO @afiune Move or re-create the common messages inside the api/ folder
 import "api/external/common/version/version.proto";
```
Comment (msorens) [inline api/external/applications/applications.proto:11]: Totally by chance, this first file shows all 3 variations of things that were stripped via `sed`.

File: components/automate-gateway/api/api_test.go [hunk 2]
```diff
@@ -42,12 +42,10 @@ func TestGeneratedPolicyUpToDate(t *testing.T) {
 }
 
 func TestAllProtoFilesAnnotated(t *testing.T) {
```
Comment (msorens) [inline components/automate-gateway/api/api_test.go:44]: And yes, we did have a test for these!

**Comment by srenatus** [comment]
In the issue, we've discussed also removing the v1 bits of the protoc plugin. Anyhow, this could just as well be done in another PR.

**Comment by msorens** [comment]
@srenatus I assume you mean this? https://github.com/chef/automate/pull/3050/commits/fcf4e19e03d22008681fcae684f5f95d7609bd1d

**Comment by srenatus** [comment]
@msorens oh that was already done. Cool. Ignore me =)

**Pull request merged**